import-secrets/handler.go

```diff
@@ -67,12 +67,12 @@ func Handle(req []byte) string {
 		return fmt.Sprintf("invalid owner name %s", event.owner)
 	}
 
-	if strings.HasPrefix(userSecret.Metadata.Name, event.owner) == false {
+	name := strings.ToLower(fmt.Sprintf("%s", userSecret.Metadata.Name))
+
+	if strings.HasPrefix(name, strings.ToLower(event.owner)) == false {
```

Comment:
I don't think we need to toLower both parts:

```
	ownerNormalized := strings.ToLower(event.owner)

	if strings.HasPrefix(userSecret.Metadata.Name, ownerNormalized) == false {
		return fmt.Errorf("unable to bind a secret which does not start with owner name: %s", ownerNormalized).Error()
	}
```

Reply:
Fixed as well!